specification/monitor/resource-manager/Microsoft.Insights/stable/2015-04-01/autoscale_API.json

```diff
@@ -803,6 +814,39 @@
           "type": "string"
         }
       }
+    },
+    "ScaleRuleMetricDimension": {
+      "required": [
+        "DimensionName",
+        "Operator",
+        "Values"
+      ],
+      "properties": {
+        "DimensionName": {
+          "type": "string",
+          "description": "Name of the dimension."
+        },
+        "Operator": {
+          "type": "string",
+          "description": "the dimension operator. Only 'Equals' and 'NotEquals' are supported. 'Equals' being equal to any of the values. 'NotEquals' being not equal to all of the values",
+          "enum": [
+            "Equals",
+            "NotEquals"
+          ],
+          "x-ms-enum": {
+            "name": "ScaleRuleMetricDimensionOperationType",
+            "modelAsString": false
```

Comment:
why this is false?

Reply:
As talked offline, change this value to true to avoid breaking change